lib/ember-qunit/qunit-module.js

```diff
@@ -33,9 +33,13 @@ export function createModule(Constructor, name, description, callbacks) {
   var beforeEach = beforeEachCallback(callbacks || description);
   var afterEach  = afterEachCallback(callbacks || description);
 
-  var module = new Constructor(name, description, callbacks);
+  var module;
 
   qunitModule(module.name, {
+    before() {
+      module = new Constructor(name, description, callbacks);
```

Comment:
Can we add a comment that we don't want to assign this on `this` to avoid allowing the test to access it?